run.sh

```diff
@@ -1,31 +1,77 @@
-#!/bin/sh
+#!/bin/bash
 
-while getopts ":u:p:s:" opt; do
-  case $opt in
-    u) D_USERNAME="$OPTARG"
-    ;;
-    p) D_PASSWORD="$OPTARG"
-    ;;
-    s) D_SHELL="$OPTARG"
-    ;;
-    \?) echo "Invalid option -$OPTARG" >&2
-    ;;
+USERNAME=user
+PASSWORD=password
+SHELL=bash
+RUN_ARGS="-d"
+MOUNT_LOCAL=0
+FORCE_REBUILD=0
+
+# HELP, I NEED SOMEBODY
+showHelp() {
+  echo "Usage: $(basename $0) [-u username] [-p password] [-s shell] [-i] [-m] [-f]"
+  echo ""
+  echo "  Use -u, -p, and -s to set up your environment. In addition:"
+  echo ""
+  echo "  -i  Launch the container interactively and open the shell prompt."
+  echo "      Otherwise, the container will run in daemon mode."
+  echo "  -m  Mount the .ssh and .aws directories from the host as volumes in"
```

Comment:
I like this, but I wonder if we're being too restrictive with these specifics. What if `-m` accepted a comma-delimited set of paths?

Reply:
Good call. I was erring on the side of simpler arguments (since, at a certain point, you might as well just be typing a docker build command yourself), but I think expressive-as-possible makes more sense since we'll probably want to alias our config for this on a per-env basis anyway.

To simplify, though, rather than do string processing in bash, I'll just allow for multiple `-m` arguments to be passed, relative to $HOME. So the current functionality would be replicated with `-m .ssh -m .aws`.